src/api/db/migrate/20171019101029_disallow_null_for_group_id_in_groups_users.rb

```diff
@@ -0,0 +1,11 @@
+class DisallowNullForGroupIdInGroupsUsers < ActiveRecord::Migration[5.1]
+  def up
+    # Remove all dangling groups users that got created since 18680d611347d312
+    GroupsUser.where(group_id: nil).destroy_all
```

Comment:
I am split about this one... I guess it's consistent with what we already do. But if you know a better way to handle this, please let me know.